lib/migrate.js

```diff
@@ -0,0 +1,207 @@
+// Copyright IBM Corp. 2017,2019. All Rights Reserved.
+// Node module: loopback-connector-cloudant
+// This file is licensed under the Apache License 2.0.
+// License text available at https://opensource.org/licenses/Apache-2.0
+
+'use strict';
+
+const _ = require('lodash');
+const inspect = require('util').inspect;
+const debug = require('debug')('loopback:connector:cloudant');
+
+module.exports = mixinMigrate;
+
+function mixinMigrate(Cloudant) {
+  /**
+    * Used in function `migrateOrUpdateIndex`.
+    * Create index with index object.
+    * @param {Object} mo  The model configuration
+    * @param {String} model  The model name.
+    * @param {String} name  The index name.
+    * @param {Object} indexObj The index object
+    * e.g. {fields: [{"foo": "asc"}, {"bar": "asc"}], partitioned: true}
+    * @param {Function} cb
+  */
+  Cloudant.prototype.__createIndex = function(mo, model, name, indexObj, cb) {
```

Comment:
similar to [comment](https://github.com/strongloop/loopback-connector-couchdb2/pull/69#discussion_r345284015) from @bajtos, it should probably be single `_` in the function name.

Reply:
👍 renamed them to single `_`